packages/dashboard/src/components/queryEditor/__tests__/advancedSearch.spex.tsx

```diff
@@ -0,0 +1,471 @@
+import { QueryEditor } from '../queryEditor';
+import { type IoTSiteWiseClient } from '@aws-sdk/client-iotsitewise';
+import React from 'react';
+import { render, screen, waitFor, fireEvent } from '@testing-library/react';
```

Comment:
Why not use '@testing-library/user-event' for testing events? I believe I am using this in the other tests.

Reply:
I didn't know about `@testing-library/user-event` and found `fireEvent` from the `@testing-library/react` docs. I can switch to `user-event`, what's the benefit?